Source/WebService/Blackjack.WebService/ClientApp/src/app/components/play-game-page/play-game.component.ts

```diff
@@ -203,30 +203,32 @@ export class PlayGameComponent implements OnInit {
   /**
    * Resolves any situation in which the player wins a hand of blackjack
    */
-  public winOutcome(): void {
+  public async winOutcome(): Promise<void> {
     this.user.balance += (this.wager * 2);
     this.wager = 0;
-    this._userApi.updateUser(this.user);
+    await this._userApi.updateUser(this.user);
   }
 
   /**
    * Resolves any situation in which the player loses a hand of blackjack
-   */  public loseOutcome(): void {
+   */
+    public async loseOutcome(): Promise<void> {
     this.wager = 0;
-    this._userApi.updateUser(this.user);
+    await this._userApi.updateUser(this.user);
   }
 
   /**
    * Resolves any situation in which the player ties in a hand of blackjack
    */
-  public pushOutcome(): void {
+  public async pushOutcome(): Promise<void> {
     this.user.balance += this.wager;
     this.wager = 0;
+    await this._userApi.updateUser(this.user);    
   }
 
   public async ngOnInit(): Promise<void> {
     this.user = new UserDto();
-    this.user = await this._userApi.getUser('test');
+    this.user = await this._userApi.getUserByID(1);
```

Comment:
Keep in mind this only works because you as the dev know you only have one user.

If this app supported hundred or thousands of player this value would need to be programmatically supplied.

That can be built out after this, but I just wanted to make sure to explicitly call out that this is a stop-gap solution that enables you to test/run your code locally, but is not the final implementation.

Reply:
Yeah I figured it would change when the login was implemented. I'm assuming this.user wouldn't be declared as new but instead be something like 

this.user = await this._userApi.getUser(userID);

where userID is passed by the login page or whatever page calls this one. If I recall correctly, you said you dont want to be passing around entire objects between pages, right?

Reply:
Yee, that's right. It'll be more apparent when that work starts, but just wanted to make sure that was clear during this interim period.